modules/tinymce/src/core/main/ts/textpatterns/utils/Utils.ts

```diff
@@ -43,7 +43,6 @@ const resolveFromDynamicPatterns = (patternSet: PatternSet, block: Element, befo
     text: beforeText,
     block
   });
-
```

Comment:
This line should not have been deleted... please add it back as it makes it harder to read without this separator.